composer.json

```diff
@@ -24,39 +24,40 @@
         "az-digital/arizona-bootstrap": "2.0.5",
         "composer/installers": "^1.9",
         "cweagans/composer-patches": "^1.7",
+        "drupal/auto_entitylabel": "3.0-beta3",
         "drupal/background_image_formatter": "1.7",
         "drupal/bootstrap_barrio": "4.30",
         "drupal/cas": "1.7",
+        "drupal/ckeditor_bs_grid": "2.0.3",
+        "drupal/coffee": "1.0",
         "drupal/config_distro": "1.0-alpha4",
         "drupal/config_provider": "2.0-rc4",
         "drupal/config_snapshot": "1.0-rc2",
         "drupal/config_sync": "2.0-beta7",
         "drupal/config_update": "1.7",
         "drupal/core-recommended": "9.1.0",
         "drupal/ctools": "3.4",
+        "drupal/date_ap_style": "^1.0",
+        "drupal/draggableviews": "2.0.0",
         "drupal/easy_breadcrumb": "1.13",
+        "drupal/embed": "1.4",
+        "drupal/entity_embed": "1.1",
         "drupal/externalauth": "1.3",
         "drupal/field_group": "3.1",
+        "drupal/field_group_link": "^3.0@RC",
+        "drupal/media_library_form_element": "2.0",
         "drupal/media_library_theme_reset": "1.0",
+        "drupal/metatag": "1.15",
         "drupal/migrate_plus": "5.1",
         "drupal/migrate_tools": "5.0",
-        "drupal/pathauto": "1.8",
-        "drupal/token": "1.7",
```

Comment:
The sorting changes in this file may cause conflicts with other open PRs so we might want to consider merging this PR last today.